### PR TITLE
[Coggers] Adds COG-Nac

### PR DESCRIPTION
**Pull request by redmoogle**
# Credits to: UselessTheremin

I just reopened it with the fixes

# General Documentation

### Intent of your Pull Request

Adds cognac which makes you spin every sip
Liquid of the justicars

### Why is this change good for the game?

It's a cool drink

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds a new drink called COG-Nac which makes you spin each sip

### What should players be aware of when it comes to the changes your PR is implementing?
Coggers

### What general grouping does this PR fall under? 
Bar Buff

# Changelog

:cl:  
rscadd: Adds COG-Nac, liquid of the justicars
/:cl:

### CODE REVIEW

**Comment by Reed0506** [comment]
give it rotatium effects or something because funny speen

**Review by JamieD1 — APPROVED** [review]
